Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -151,14 +151,30 @@ def bump
   end
 
   def livecheck_result(formula_or_cask)
-    skip_result = Livecheck::SkipConditions.skip_information(formula_or_cask)
-    if skip_result.present?
-      return "#{skip_result[:status]}#{" - #{skip_result[:messages].join(", ")}" if skip_result[:messages].present?}"
+    name = Livecheck.formula_or_cask_name(formula_or_cask)
+
+    livecheck_formula_or_cask, =
+      Livecheck.resolve_livecheck_reference(formula_or_cask, full_name: false, debug: false)
+
+    # Check skip conditions for a referenced formula/cask
+    if livecheck_formula_or_cask
+      skip_info = Livecheck::SkipConditions.referenced_skip_information(
+        livecheck_formula_or_cask,
+        name,
+        full_name: false,
+        verbose:   false,
+      )
+    end
+
+    skip_info ||= Livecheck::SkipConditions.skip_information(formula_or_cask, full_name: false, verbose: false)
+    if skip_info.present?
+      return "#{skip_info[:status]}#{" - #{skip_info[:messages].join(", ")}" if skip_info[:messages].present?}"
     end
 
     version_info = Livecheck.latest_version(
       formula_or_cask,
-      json: true, full_name: false, verbose: false, debug: false,
+      livecheck_formula_or_cask: livecheck_formula_or_cask,
+      json: true, full_name: false, verbose: false, debug: false
```

Comment:
With this argument format (not all on their own lines), `brew style` complains if there's a trailing comma and I had to remove it for this to pass. If all of the arguments were on separate lines, it would want the trailing comma but then you have to align the hash values and it turns into a bit of a monstrosity:

```ruby
version_info = latest_version(
  formula_or_cask,
  livecheck_formula_or_cask: livecheck_formula_or_cask,
  livecheck_references:      livecheck_references,
  json:                      json,
  full_name:                 use_full_name,
  verbose:                   verbose,
  debug:                     debug,
)
```

Having the repeated CLI arg values all on one line keeps this a little more manageable in my opinion.